galley/pkg/config/analysis/analyzers/util/config.go

```diff
@@ -37,7 +38,7 @@ var (
 
 // IsSystemNamespace returns true for system namespaces
 func IsSystemNamespace(ns resource.Namespace) bool {
```

Comment:
IsSystemNamespace is used by bug report as well

Reply:
@howardjohn Yes, but it is only used to skip running analysis!  This change will just make it skip _istio-system_, which shouldn't be analyzed anyway.